src/tools/genpolicy/rules.rego

```diff
@@ -339,6 +353,7 @@ allow_linux(p_oci, i_oci) {
 
     allow_masked_paths(p_oci, i_oci)
     allow_readonly_paths(p_oci, i_oci)
+    allow_linux_devices(p_oci.Linux.Devices, i_oci.Linux.Devices)
 
     print("allow_linux: true")
```

Comment:
I would also add:

count(i_oci.Linux.Resources.Devices) == 0

But we could try this in a separate PR.